README.md

```diff
@@ -25,8 +25,15 @@ npm i
 
 ## Usage
 
+To generate test data used in the test suite, testers are required to specify
+the issuer name using the environment variable `ISSUER_NAME`.
+
+Additionally, vc holder name may also be specified for generating disclosed
+test credentials for ecdsa sd tests using the environment variable
+`HOLDER_NAME`. If it is not specified, `Digital Bazaar` vc holder will be used.
```

Comment:
```suggestion
In addition, VC holder name for generating disclosed test credentials for
ECDSA SD tests may be specified using the environment variable `HOLDER_NAME`.
If `$HOLDER_NAME` is not specified, `Digital Bazaar` will be used.
```

Reply:
We may want to start being consistent with using `ECDSA-SD` as well.